sdk/identity/identity/src/credentials/managedIdentityCredential/fabricMsi.ts

```diff
@@ -56,6 +57,9 @@ function prepareRequestOptions(
   if (clientId) {
     queryParameters.client_id = clientId;
   }
+  if (resourceId) {
+    queryParameters.mi_res_id = resourceId;
```

Comment:
Have you tried this? The Service Fabric documentation doesn't give any information about specifying user assigned identities in token requests.

Reply:
Ohh thanks! I’ll skip it for know. How do you know these things? 🙂 I’ve got to get better at checking the API docs.